metricbeat/module/system/filesystem/helper.go

```diff
@@ -137,18 +130,20 @@ func AddFileSystemUsedPercentage(f *FSStat) {
 }
 
 // GetFilesystemEvent turns a stat struct into a MapStr
-func GetFilesystemEvent(fsStat *FSStat) common.MapStr {
+func GetFilesystemEvent(fsStat *FSStat, addStats bool) common.MapStr {
 	evt := common.MapStr{
 		"type":        fsStat.SysTypeName,
 		"device_name": fsStat.DevName,
 		"mount_point": fsStat.Mount,
-		"total":       fsStat.Total,
-		"available":   fsStat.Avail,
-		"free":        fsStat.Free,
-		"used": common.MapStr{
+	}
+	if addStats == true {
```

Comment:
Nit. Redundant boolean comparison.
```suggestion
	if addStats {
```